tools/integration_tests/emulator_tests/write_stall_test.go

```diff
@@ -88,3 +88,90 @@ func TestChunkTransferTimeoutInfinity(t *testing.T) {
 		test_setup.RunTests(t, ts)
 	}
 }
+
+// Test suite for chunk transfer timeouts.
+type chunkTransferTimeoutSuite struct {
+	flags []string
+	// Timeout for chunk transfers, in seconds.
+	chunkTransferTimeoutSecs int
+}
+
+// Setup for each test case.
+func (s *chunkTransferTimeoutSuite) Setup(t *testing.T) {
+}
+
+// Teardown for each test case.
+func (s *chunkTransferTimeoutSuite) Teardown(t *testing.T) {
+	setup.UnmountGCSFuse(rootDir)
+	assert.NoError(t, emulator_tests.KillProxyServerProcess(port))
+}
+
+// Test case: single write stall.
+func (s *chunkTransferTimeoutSuite) TestHandlesWriteStalls(t *testing.T) {
+	configPath := "./proxy_server/configs/write_stall_40s.yaml"
```

Comment:
code is duplicated . can you use t.run here also?

Reply:
This configuration file related to the test itself, not to flag sets. For example, it enforces a one stall, and both flag sets execute the test with this configuration.

Reply:
Updated as discussed offline.